### ISSUE TITLE
"Later" button on MixedRealityToolkit object missing dialog is non-functional

### ISSUE THREAD

**Issue by davidkline-ms** [microsoft/MixedRealityToolkit-Unity#3526]
If you do not have the MixedRealityToolkit object in your scene and you attempt to interact with an MRTK object you will get the following dialog:

![errordialog](https://user-images.githubusercontent.com/13281406/53393100-4340d500-3950-11e9-8456-b4551cc5d501.png)

Clicking the Later button does nothing. The button should honor the customer's desire to dismiss without changes.

Repro steps:
* Configure scene for MRTK
* Delete MixedRealityToolkit object
* Edit > Preferences
* Change a setting
* Click "Later"

**Comment by SimonDarksideJ** [comment]
Confirmed, Later is being called repeatedly and MRTK is not respecting that.

This is compounded by the original replication steps I've added to #3565 where by Unity is preserving the MR Manager even if it's not in the scene once run, it's not being cleared down appropriately.

So 2 issues:
* This issue on multiple warnings being presented
* #3565 where MRTK is being preserved in Unity memory between scenes.
